APIManager.js

```diff
@@ -0,0 +1,67 @@
+class APIManager {
+    constructor(renderer) {
+        this.renderer = renderer
+    }
+
+    getUsers() {
+        $.ajax({
+            method: `GET`,
+            url: `https://randomuser.me/api/?results=7`,
+            dataType: 'json',
+            success: data => {
+            console.log(data)
+                this.renderer.renderUsers(data)
```

Comment:
`this.data.user.firstName = data.blahblah`

store the data gathered from each API into `this.data` in the constructor